samples/tetris/build.gradle

```diff
@@ -104,12 +112,27 @@ afterEvaluate {
 }
 
 private def determinePreset() {
-    def preset = isRaspberryPiBuild() ? kotlin.presets.linuxArm32Hfp /* aka RaspberryPi */ : MPPTools.defaultHostPreset(project)
+    def preset = null
+    if (isRaspberryPiBuild()) {
+        preset = kotlin.presets.linuxArm32Hfp // aka RaspberryPi
+    } else if (isMingwX86Build()) {
+        preset = kotlin.presets.mingwX86
+    } else {
+        preset = MPPTools.defaultHostPreset(project)
+    }
+    assert preset != null
+    
     println("$project has been configured for ${preset.name} platform.")
     preset
 }
 
 // If host platform is Linux and RaspberryPi target is activated.
 private boolean isRaspberryPiBuild() {
     MPPTools.isLinux() && Boolean.parseBoolean(project.findProperty('tetris.raspberrypi.build') as String)
-}
\ No newline at end of file
+}
+
+private boolean isMingwX86Build() {
+    MPPTools.isWindows() && (
+            System.getenv("ProgramFiles(x86)") == null 
```

Comment:
Why? Host must be 64-bit, so here cannot be null.

Reply:
32-bit host does not have `ProgramFiles(x86)` environment variable. That's why 32-bit tetris will be built under 32-bit host. What's not correct here?

Reply:
32-bit Windows hosts cannot run Kotlin/Native compiler.

Reply:
Kotlin/Native does not support 32-bit hosts.

Reply:
OK